Add X-KDE-SubstituteUID=true to the mdmsetup desktop file to hide the entry
for non-admin users in KDE.

Index: mdm-2.18.1/gui/mdmsetup.desktop.in.in
===================================================================
--- mdm-2.18.1.orig/gui/mdmsetup.desktop.in.in	2007-05-02 17:47:25.000000000 +0200
+++ mdm-2.18.1/gui/mdmsetup.desktop.in.in	2007-05-02 17:47:41.000000000 +0200
@@ -9,6 +9,7 @@
 Terminal=false
 Type=Application
 Categories=GNOME;GTK;Settings;System;
+X-KDE-SubstituteUID=true
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=mdm
 X-GNOME-Bugzilla-Component=general
